receiver/sqlqueryreceiver/testdata/integration/sqlserver/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+#
+#
+
+
+# Start SQL Server in the background
+/opt/mssql/bin/sqlservr &
+
+# Wait for SQL Server to start
+sleep 30s
```

Comment:
Is there any way to have a more concrete/time sensitive check for this?

Reply:
Yes, I removed hardcoded 30s sleep, and changed it to the [check every 1s if db is available](https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/b0efb0d847e5627aa79263da9b60b576d1481e7e#diff-4eadc3f91ac318e7d410103d2aae66dd174aec0369dfa0974acb16072c0b325cR13)